source/store/CaseContext.js

```diff
@@ -90,7 +89,7 @@ export function CaseProvider({ children }) {
    * Function for sending a put request towards the case api endpoint, updating the currently active case
    * @param {obj} data a object consiting of case user inputs.
    */
-  const updateCurrentCase = (data, status, currentStep) => {
+  const updateCurrentCase = async (data, status, currentStep) => {
```

Comment:
Might be a good idea to add a try catch in this function in case the put request fails.